.github/workflows/client-go.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2-
       - name: Compile IoTDB Server
-        run: mvn -B package -Dmaven.test.skip=true  -am -pl server
```

Comment:
Hi, I don't think using maven-wrapper in all CI workflows is a good idea. It seems take more time than preinstalled maven.

Reply:
revert.It looks like it is.